posts/2019/05/quansight-partners-with-quansight-to-sponsor-spyder.md

```diff
@@ -0,0 +1,112 @@
+<!--
+.. title: Quansight Partners with Quansight to sponsor Spyder
+.. slug: quansight-partners-with-quansight-to-sponsor-spyder
+.. date: 2019-05-16 21:22:58 UTC-05:00
+.. tags: Spyder
+.. category: 
+.. link: 
+.. author: Gonzalo Peña-Castellanos
+.. description: 
+.. type: text
+-->
+
+TDK-Micronas is sponsoring Spyder development efforts through Quansight Labs.
```

Comment:
Should we clarify Quansight Labs role a bit at this point? I mean, what role does it have between TDK and Spyder?

Reply:
Yes, I think that would be quite useful. Do you want me to write something and you review/tweak, or the other way around?

Reply:
I think it'd be better if you write it instead of us.

Reply:
Sounds good, I'll give it a try on the weekend or Monday. We have some time, this is the second blog post and I want to space them apart a little for better exposure.

Reply:
I propose to insert the following text at the end of this paragraph:

At this point it may be useful to clarify the relationship the role of Quansight Labs in Spyder's development and the relationship with TDK. To quote Ralf Gommers (director of Quansight Labs):

_We're an R&D lab for open source development of core technologies around data science and scientific computing in Python. And focused on growing communities around those technologies. That's how I see it for Spyder as well: Quansight Labs enables developers to be employed to work on Spyder, and helps with connecting them to developers of other projects in similar situations. Labs should be an enabler to let the Spyder project, its community and individual developers grow. And Labs provides mechanisms to attract and coordinate funding. Of course the project is still independent. If there are other funding sources, e.g. donations from individuals to Spyder via OpenCollective, all the better._